tests/fixtures.py

```diff
@@ -365,7 +403,15 @@ def simple_http_fixture(httpserver):
         query_string='optType=ReadRealTimeData',
         response=X1_BOOST_AIR_MINI_RESPONSE,
         inverter=inverter.X1Mini,
-        values=X1_MINI_VALUES,
+        values=X1_MINI_BOOST_AIR_VALUES,
```

Comment:
This library needs to support both the existing X1 mini values _plus_ the new V34 X1 mini values